app/src/main/java/org/dhis2/usescases/login/LoginViewModel.kt

```diff
@@ -178,11 +178,16 @@ class LoginViewModel(
     }
 
     fun onLoginButtonClick() {
-        view.hideKeyboard()
-        analyticsHelper.setEvent(LOGIN, CLICK, LOGIN)
-        increment()
-        logIn()
-        decrement()
+        try {
```

Comment:
Why don't you create a couple of tests for this?. For success and error case.

Reply:
Yes, some were already active but I have linked the other issue we had for the ignored ones.

Reply:
I was thinking about the LoginViewModel as a unit tests